Rakefile

```diff
@@ -11,7 +11,8 @@ Vmdb::Application.load_tasks
 begin
   require 'jasmine'
   load 'jasmine/tasks/jasmine.rake'
-rescue LoadError
+rescue LoadError => e
+  puts e.message
```

Comment:
But, in those environments, we don't want output, I believe, so maybe just doing nothing in the rescue is fine.